src/ring_theory/power_series/basic.lean

```diff
@@ -1813,10 +1821,47 @@ begin
     exact ih t.2 },
 end
 
+-- TODO: link with `X_pow_dvd_iff`
```

Comment:
isn't this easy with `coeff_of_lt_order`? or is there more to it?

Reply:
`X_pow_dvd_iff` requires commutativity. It might not need to (my baseless intuition says it shouldn't), maybe when it was defined, noncommutativity wasn't a target.